curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/67460b84234b70a4772d2094.md

```diff
@@ -0,0 +1,93 @@
+---
+id: 67460b84234b70a4772d2094
+title: Task 33
+challengeType: 19
+dashedName: task-33
+---
+
+<!-- (Audio) James: A bit, but solving problems like these is what makes our job interesting, right? -->
+
+<!-- SPEAKING -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+How could Lisa confirm in a single word and enthusiastically that she agrees with what James just said?
+
+## --answers--
+
+`Absolutely!`
+
+---
+
+`Objection!`
+
+### --feedback--
+
+`Objection` is normally used in court when an attorney says they disagree with what has been said or proposed by another attorney. It expresses disagreement, not agreement.
+
+## --video-solution--
+
+1
+
+
+# --explanation--
+
+The word `absolutely` is used to express complete agreement or certainty. It is often used in informal or conversational contexts to emphasize strong agreement.  
+
+Example: - `Do you think we should move forward with this plan?` - `Absolutely!` (in this case, `Absolutely` shows that the speaker is fully in favor of the plan and has no doubts)
+
+In this dialog, Lisa is strongly agreeing with James’s point about their job being interesting because of the challenges they solve.
```

Comment:
```suggestion
`Absolutely` is used to express complete agreement or certainty. It is often used in informal or conversational contexts to emphasize strong agreement.  For example: 

Person 1: `Do you think we should move forward with this plan?` 

Person 2: `Absolutely!` -  Here, `Absolutely` shows that the speaker is fully in favor of the plan and has no doubts.

In this dialog, Lisa is strongly agreeing with James’s point about their job being interesting because of the challenges they solve.
```